src/database/test/DatabaseTests.cpp

```diff
@@ -377,13 +377,56 @@ class SchemaUpgradeTestApplication : public TestApplication
 
 TEST_CASE("schema upgrade test", "[db]")
 {
-    auto test_one_db_mode = [](Config::TestDbMode const db_mode) {
+    auto prepOldSchemaDB = [](SchemaUpgradeTestApplication& app) {
+        auto& session = app.getDatabase().getSession();
+        {
+            std::string const mAccountID = "account";
```

Comment:
Nitpick: code-style conventions again, local variables are camel-case with first letter lower.

Reply:
Yeah, I caught myself there in a later commit too (yet another bit of review I could have saved you by squashing commits).

Reply:
Caught myself there too.  :)